test/unit/js/flicking.test.js

```diff
@@ -44,757 +61,614 @@ function simulator(el, option, callback) {
 	}, option), callback);
 }
 
+// hooks for each test
 var hooks = {
 	beforeEach: function() {
-		this.inst = null;
+		this.inst = [];
+		this.create = window.create;
 	},
 	afterEach: function() {
-		this.inst && this.inst._mcInst.destroy();
+		$.each($.makeArray(this.inst), function(i, v) {
```

Comment:
this.inst could be array or instance object.
If iterate in that way, in case of instance will iterate over object members.

It can be added a condition to determine data type of this.inst, but just using .makeArray is more simple than that. What do you think?